### DISCUSSION TITLE
Since there is the parameter 'sys.files' in the workflow setting, A PDF file can be uploaded during the chatflow?

### DISCUSSION THREAD

**Discussion by WSDzju** [langgenius/dify#4233] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Pleas do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing?

Yes, when trying to perocess a PDF with images, it is impossible to chat with the PDF

### 2. Describe the feature you'd like to see

PDF,DOC, and Pic can be uploaded during the chat

### 3. How will this feature improve your workflow or experience?

improve the coverage of functions

### 5. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by crazywoola** [comment]
In chat doc upload is still unavailable for current versions. So the answer is no.

**Reply by XinyueZ** [reply]
Is there solution or workaround?

**Comment by wwjCMP** [comment]
> In chat doc upload is still unavailable for current versions. So the answer is no.

I hope this feature can be implemented, it would be very useful.
